foodsaving/conversations/models.py

```diff
@@ -97,6 +97,22 @@ def type(self):
 
         return type
 
+    def find_group(self):
+        if self.is_private:
+            return None
+        if self.target_type_id is None:
+            return None
+
+        type = self.type()
+        if type == 'pickup':
```

Comment:
I once had something like this implemented the other way round:
Each object would specify a property that refers to the group, e.g. `pickup.group` would return the group. The group itself also has a `group` property.
Therefore this code could be simplified to:

```
return self.target.group
```

Reply:
It's partially implemented, here's an example: https://github.com/yunity/karrot-backend/blob/e5c048925ca24b3e99de1a96d76c0932a50240ad/foodsaving/pickups/models.py#L234-L236